backend/src/schema/resolvers/reports.js

```diff
@@ -38,12 +29,12 @@ export default {
           },
         )
         log(reportTransactionResponse)
-        return reportTransactionResponse.records.map(transformReturnType)
+        return reportTransactionResponse.records.map(record => record.get('returnedResource'))
       })
       try {
-        const [createdRelationshipWithNestedAttributes] = await reportWriteTxResultPromise
-        if (!createdRelationshipWithNestedAttributes) return null
-        return createdRelationshipWithNestedAttributes
+        const [reportedResource] = await reportWriteTxResultPromise
+        if (!reportedResource) return null
+        return reportedResource
```

Comment:
```suggestion
        return reportedResource || null
```
If you want to convert `undefined` to `null`.

Reply:
that makes sense!

Reply:
Yes, I like this better! 😍 @roschaefer